src/account.ts

```diff
@@ -24,11 +24,10 @@ export async function getWallet() {
     return _cachedWallet;
   }
 
-  let mnemonic = await KeyManager.getMnemonic();
+  const mnemonic = await KeyManager.getMnemonic();
 
   if (!mnemonic) {
-    mnemonic = await KeyManager.generateMnemonic();
-    await KeyManager.saveMnemonic(mnemonic);
```

Comment:
We'll need to persist the mnemonic elsewhere (like after line 13)

Reply:
Ohh jeez, that's hilarious. So `createAccount` was never actually creating a true mobile account, because we accidentally removed key persistence. Which would also explain why I was so confused that the key was changing on me between the call to `createAccount` and `getAccount` with react reload.